server/manager/src/main/java/org/apache/accumulo/manager/tableOps/tableImport/PopulateMetadataTable.java

```diff
@@ -121,63 +114,56 @@ static void readMappingFile(VolumeManager fs, ImportedTableInfo tableInfo, Strin
           Text currentRow = null;
           int dirCount = 0;
 
-          while (true) {
-            key.readFields(in);
-            val.readFields(in);
+          try (DataInputStream in = new DataInputStream(new BufferedInputStream(zis))) {
```

Comment:
Seems like this could close zis, which would not be desired inside the loop.    I am uncertain about the actual behavior though.